internal/storage/elasticsearch/dbmodel/model.go

```diff
@@ -54,6 +54,7 @@ type Span struct {
 	// so Jaeger maps StartTime to a 'long' type. This extra StartTimeMillis field
 	// works around this issue, enabling timerange queries.
 	StartTimeMillis uint64     `json:"startTimeMillis"`
+	SpanKind        string     `json:"spanKind,omitempty"`
```

Comment:
* odd position between start time and duration fields
* while at it I would add span status too, it's the only other field that makes sense to be at the top level